frontend/guardian.rb

```diff
@@ -29,31 +28,24 @@ def restart(command)
   return tag_reporter
 end
 
-def close_door(gpio_command)
-  `#{gpio_command} -g write 9 1`
-end
-
-def open_door(gpio_command)
-  `#{gpio_command} -g write 9 0`
+def close_door(gpio_command, door)
```

Comment:
Makes sense, but I'm struggling to come up with a name that's not awful like `stop_opening_door`. Any suggestions?